app/controllers/cars_controller.rb

```diff
@@ -19,7 +19,7 @@ def index
   # GET /cars/rookie or /cars/rookie.json
   def rookie
     @cars = Rails.cache.fetch('rookie_cars', :expires_in => 1.month) do
-      @cars = cars_of_category(SYS::CATEGORY::ROOKIE)
+      @cars = cars_of_category(SYS::CATEGORY::ROOKIE).sort_by { |car| [car.multiplier, car.stock? ? 0 : 1] }
```

Comment:
Would it be possible to add another condition that sort the cars by name too? After searching a bit I think it might be something like `.sort_by { |car| [car.multiplier, car.name, car.stock? ? 0 : 1] }`. (Reminder that I'm not a coder lol).
It doesn't really matter if stock cars lose the order they have between themselves, which is not by alphabet but by how the appear in-game.

Reply:
Done! Thank you and sorry the delay